crypto/stark/src/traits.rs

```diff
@@ -62,9 +62,9 @@ pub(crate) mod tests {
 
     #[derive(Clone, PartialEq, Debug)]
     pub(crate) struct Claim {
-        index:    usize,
-        value:    FieldElement,
-        exponent: usize,
+        index:            usize,
+        pub(crate) value: FieldElement,
```

Comment:
Hmm.. we should really use this struct outside of this module. But this probably requires a refactor of the test constraints. Out of scope for this PR.